### PR TITLE
Dbex/58558: EVSS Migration - rated disabilities - implement `rated_disability_id` in Lighthouse API Provider transform layer

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- map rated_disabilities_id in api response in Lighthouse RatedDisabilityProvider transform layer
- update unit tests to not use rated_disability_id of "0" for more accurate testing of method logic
- makes client_id and rsa_key_path optional for veteran verification service calls
- cleans up where we were duplicating the use of the veteran verification form526 credentials block
- 
## Related issue(s)
- department-of-veterans-affairs/va.gov-team#58558
- #58558

## Testing done

- unit testing
- to test the default LH credentials, assuming the settings file prepared correctly with the lighthouse.veteran_verification.form526 block (example in settings.yml) use the console to make calls to the veteran verification LH service without providing a client_id or rsa_key_path as parameters

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
Form 21-526EZ, or anywhere that uses vets-api v0/rated_disabilities endpoint

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by tpharrison — APPROVED** [review]
Just reviewed the code as it relates to `get_combined_disability_rating`.  LGTM 👍

File: lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb [hunk 1]
```diff
@@ -39,12 +39,12 @@ def transform(data)
     rated_disabilities =
       data.map do |rated_disability|
         DisabilityCompensation::ApiProvider::RatedDisability.new(
-          name: rated_disability['diagnostic_type_name'],
+          name: rated_disability['diagnostic_text'],
           decision_code: decision_code_transform(rated_disability['decision']),
-          decision_text: rated_disability['description'],
+          decision_text: rated_disability['decision'],
```
Comment (mchae-nava) [inline lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb:44]: Nice catch!

**Review by mchae-nava — APPROVED** [review]
LGTM!

**Review by navaeddie — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
Thanks for all of the additional reviews! 🚀

**Pull request merged**